app/helpers/redirect_destination_helper.rb

```diff
@@ -1,16 +1,28 @@
 module RedirectDestinationHelper
   def redirect_destination_select_options
-    guide_select_options = Guide
-      .live
-      .order(:slug).pluck(:slug)
-
-    topic_select_options = Topic
-      .order(:path).pluck(:path)
-
     {
       "Other" => ["/service-manual", "/service-manual/service-standard"],
       "Topics" => topic_select_options,
       "Guides" => guide_select_options,
     }
   end
+
+private
+
+  def guide_select_options
+    Guide.live.order(:slug).pluck(:slug)
+  end
+
+  def topic_select_options
+    Topic.order(:path)
```

Comment:
It might be worth adding an `includes(:topic_sections)` to this; without it, we'll be making an extra query for each topic in `paths_for_topic_and_its_sections`.